test/integration/sagemaker/test_dgl.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+import sys
+import pytest
+
+from sagemaker import utils
+from sagemaker.mxnet.estimator import MXNet
+
+from test.integration import RESOURCE_PATH
+from timeout import timeout
+
+DGL_DATA_PATH = os.path.join(RESOURCE_PATH, 'dgl_gcn')
+DGL_SCRIPT_PATH = os.path.join(DGL_DATA_PATH, 'gcn.py')
+
+#@pytest.mark.skipif(sys.version_info[0] < 3, reason="DGL requires python3 or above")
```

Comment:
Why was this introduced and then commented?

Reply:
I think we should uncomment it.